pages/LearnerDashboard.html

```diff
@@ -11,7 +11,7 @@ <h2>
     Oppia's mission from the very start has been "to help anyone learn anything they want in an effective and enjoyable way". The "Learner Dashboard" has been designed and developed keeping this goal in mind. If you're a new user and you have signed in for the first time, here's how the learner dashboard would look:
   </p>
 
-  <img src="static/images/noActivityLearnerDashboard.png" style="width: 100%;">
+  <img src="static/images/noActivityLearnerDashboard.png" style="width: 100%;"alt="A heading with a blue background reading 'Learner Dashboard', below which is the Oppia-Mascot promting the user to get started, along with a green 'Browse out lessons' button in the dialog. ">
```

Comment:
Here and elsewhere: below which is --> followed by

prompting is misspelled

Browse our lessons is misspelled